spec/Dockerfile_spec.rb

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2017-present Sonatype, Inc.
```

Comment:
Generally the copyright date is consistent across the project, so this can be 2016.

Reply:
Changed to 2016 (e4188af).